src/js/config/Config.ts

```diff
@@ -37,6 +37,7 @@ interface IConfiguration {
   useUIConfigFixtures?: boolean;
   unitHealthAPIPrefix: string;
   useFixtures?: boolean;
+  isDevelopment?: boolean;
```

Comment:
i'm not sure whether graphiQL justifies the introduction of a `isDevelopment`-flag in the `Config`. i would be fine with using an env-var or other stuff that indicates that this was a quick decision. adding the notion of a `development`-version feels a little odd to me.

Reply:
Hmm, I thought of this to be way cleaner than a environment variable usage, because this is the central place we store configuration for the frontend in

Reply:
you suggest to change our usages of `process.env.NODE_ENV` throughout the code then?

i see where you're coming from, but looking at this change in isolation it appears to introduce yet another concept of figuring out whether we are in development-mode (just to determine whether graphiQL should be visible). i'd be also fine with something along the lines of `showGraphiQL`, because it's much more specific and clear.